homeassistant/components/comelit/sensor.py

```diff
@@ -164,4 +167,4 @@ def native_value(self) -> StateType:
         if (status := self._zone_object.human_status) == AlarmZoneState.UNKNOWN:
             return None
 
-        return status.value
+        return cast(StateType, status.value)
```

Comment:
What is the native typing for `status.value`?

Reply:
![image](https://github.com/user-attachments/assets/6656e034-755d-4966-b92c-f6c402a174cc)

from https://github.com/chemelli74/aiocomelit/blob/main/src/aiocomelit/const.py#L40

Reply:
Thanks! Shouldn't we type the return value as `str` then?